### PR TITLE
Use validationError classes defined in styles. This way this can be ovewriten by users

### PR DESCRIPTION
**Pull request by andresgutgon**
# What? 
This one is pretty easy. In text inputs errors can be styled. 

### Example in my code
image

But Radio buttons have a hardcoded string so is not possible. 

### Solution 
Just use the prop we have for this clase

### CODE REVIEW

File: packages/vanilla/src/controls/RadioGroup.tsx [hunk 1]
```diff
@@ -58,7 +58,7 @@ export const RadioGroup = ({
   const radioInput = useMemo(() => findStyleAsClassName(contextStyles)('control.radio.input'), [contextStyles]);
   const radioLabel = useMemo(() => findStyleAsClassName(contextStyles)('control.radio.label'), [contextStyles]);
   const isValid = errors.length === 0;
-  const divClassNames = `validation  ${isValid ? classNames.description : 'validation_error' }`;
+  const divClassNames = `validation  ${isValid ? classNames.description : classNames.validationError }`;
```
Comment (andresgutgon) [inline packages/vanilla/src/controls/RadioGroup.tsx:61]: Do the same as in [the input control](https://github.com/eclipsesource/jsonforms/blob/master/packages/vanilla/src/controls/InputControl.tsx#L70)

File: packages/vanilla/src/controls/RadioGroup.tsx [hunk 2]
```diff
@@ -58,7 +58,9 @@ export const RadioGroup = ({
   const radioInput = useMemo(() => findStyleAsClassName(contextStyles)('control.radio.input'), [contextStyles]);
   const radioLabel = useMemo(() => findStyleAsClassName(contextStyles)('control.radio.label'), [contextStyles]);
   const isValid = errors.length === 0;
-  const divClassNames = `validation  ${isValid ? classNames.description : 'validation_error' }`;
+  const divClassNames = [classNames.validation]
+    .concat(isValid ? classNames.description : classNames.validationError)
+    .join(' ');
```
Comment (andresgutgon) [inline packages/vanilla/src/controls/RadioGroup.tsx:63]: Do the same as in [the input control](https://github.com/eclipsesource/jsonforms/blob/master/packages/vanilla/src/controls/InputControl.tsx#L70)

**Comment by sdirix** [comment]
Thanks for this contribution too :wink:   We'll take a look very soon!

**Review by sdirix — APPROVED** [review]
Agreed! Thanks for the contribution!

**Pull request merged**